cmd/maya-apiserver/app/server/http.go

```diff
@@ -291,6 +293,7 @@ type HTTPCodedError interface {
 	Code() int
 }
 
+// CodedError is used to provide the Code error
```

Comment:
Can this be improved? This should talk about returning an HTTPCodedError

Reply:
Updated the comment